GPU/Common/FramebufferManagerCommon.cpp

```diff
@@ -1262,14 +1280,17 @@ bool FramebufferManagerCommon::BindFramebufferAsColorTexture(int stage, VirtualF
 	}
 }
 
-void FramebufferManagerCommon::CopyFramebufferForColorTexture(VirtualFramebuffer *dst, VirtualFramebuffer *src, int flags, int layer) {
+void FramebufferManagerCommon::CopyFramebufferForColorTexture(VirtualFramebuffer *dst, VirtualFramebuffer *src, int flags, int layer, bool *partial) {
 	int x = 0;
 	int y = 0;
 	int w = src->drawnWidth;
 	int h = src->drawnHeight;
 
+	*partial = false;
+
 	// If max is not > min, we probably could not detect it.  Skip.
 	// See the vertex decoder, where this is updated.
+	// TODO: We're currently not hitting this path in Dante. See #17032
```

Comment:
It's probably not using vertex coords that get checked, so maxU probably equals minU.  We don't currently set the range for all vertex types, just 16-bit through iirc.  This has been true forever, initially to avoid any perf impact to vertex decode for the few games this was initially helping.

-[Unknown]

Reply:
Ah, right. Possibly it would always be worth it to find the min/max in through mode, although then this change would need more sophistication. So, maybe one day :)